lib/app/resources/2015_11_20_shelf.md

```diff
@@ -0,0 +1,213 @@
+There exists many frameworks and libraries in Dart that can be used to make a REST API, to serve static files or just
+to handle Http requests. Many of those use `shelf` under the hood.
+
+The reason why I want to talk about `shelf` is pretty simple: Once you understand `shelf`, you pretty much understand the base of many other Web server oriented framework out there (Redstone.dart, mojito, bridge, etc).
+
+## What is shelf?
+
+* A web server middleware for Dart
```

Comment:
- A web server middleware for Dart
- A very flexible library that makes it easy to create and compose web servers and parts of web servers.
- A pretty simple library that exposes a great set of easy to use abstractions

Par exemple... ;)